### PR TITLE
Add Pendo.

### PR DESCRIPTION
**Pull request by beekley**
## Description

Addresses: [Analytics](https://tables.area120.google.com/table/bX_kMNBW8LA35n6l5Hz_co/row/8Ly8jZv68igbnKXifue_Ft)

Pendo supports some nice things like user surveys. This PR adds the Pendo snipped pointing to a free Pendo account for LeadOut, since BC's general Pendo account is limited to one site. If they ever upgrade plans, we can migrate to their `apiKey` in the snippet.

### New

- Pendo snippet

## Testing and Reviewing

Client builds! Time will show if the analytics data makes it to Pendo.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -13,6 +14,37 @@
 
       gtag('config', 'UA-146085874-1');
     </script>
+    <!-- LeadOut-specific Pendo -->
+    <script>
+      (function (apiKey) {
+        (function (p, e, n, d, o) {
+          var v, w, x, y, z;
+          o = p[d] = p[d] || {};
```
Comment (kailamjeter) [inline client/public/index.html:22]: can you add a comment to this block? looks strange with all of the single letter variables and very unclear whats going on

File: client/public/index.html [hunk 2]
```diff
@@ -13,6 +14,37 @@
...
+          z.parentNode.insertBefore(y, z);
+        })(window, document, 'script', 'pendo');
+
+        // This function creates anonymous visitor IDs in Pendo with placeholder values, since
+        // LeadOut does not have account or user IDs.
+        pendo.initialize({
+          visitor: { id: 'VISITOR-UNIQUE-ID' },
+          account: { id: 'ACCOUNT-UNIQUE-ID' },
+        });
+      })('e7ae8c80-998b-4b63-6582-3189375da42f');
```
Comment (justinebreuch) [inline client/public/index.html:46]: Is this the API key? e7ae8c80-998b-4b63-6582-3189375da42f. If so can this be a constant?

File: client/public/index.html [hunk 3]
```diff
@@ -13,6 +14,37 @@
...
+          o._q = o._q || [];
+          v = ['initialize', 'identify', 'updateOptions', 'pageLoad', 'track'];
+          for (w = 0, x = v.length; w < x; ++w)
+            (function (m) {
+              o[m] =
+                o[m] ||
+                function () {
+                  o._q[m === v[0] ? 'unshift' : 'push']([m].concat([].slice.call(arguments, 0)));
+                };
+            })(v[w]);
```
Comment (justinebreuch) [inline client/public/index.html:32]: What does this do?

**Comment by beekley** [comment]
I closed the comments about the snippet itself, which we just copy from our Pendo account's installation page. I've added a comment that includes the source of this code (which is gated behind a login).

**Review by kailamjeter — APPROVED** [review]

**Pull request merged**